.github/workflows/mobile-android_build.yml

```diff
@@ -86,7 +86,14 @@ jobs:
           adb shell am start -n io.envoyproxy.envoymobile.helloenvoy/.MainActivity
     - name: 'Check connectivity'
       if: steps.should_run.outputs.run_ci_job == 'true'
-      run: adb logcat -e "received headers with status 301" -m 1
+      run: |
+        timeout 30 adb logcat -e "received headers with status 301" -m 1 || {
```

Comment:
if this does what I think it does this is AWESOME
cc @ravenblackx because I think it solves the issue I was talking to you about re logs

Reply:
i believe it will - with the caveat that if it was timing out on original logcat then it may still do, and it may spew a rainforest of logs